core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -712,6 +746,184 @@ export class LoggedInUser extends BaseUser {
+      explorationIdElement,
+      element => element.textContent
+    );
+    return explorationIdUrl;
+  }
+
+  /**
+   * Function for navigating to the profile page for a given username.
+   */
+  async navigateToProfilePage(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+    if (this.page.url() === profilePageUrl) {
+      return;
+    }
+    await this.goto(profilePageUrl);
+  }
+
+  /**
+   * Function to subscribe to a creator with the given username.
+   */
+  async subscribeToCreator(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+
+    if (this.page.url() !== profilePageUrl) {
+      await this.navigateToProfilePage(username);
+    }
+
+    await this.clickOn(subscribeButton);
+    await this.page.waitForSelector(unsubscribeLabel);
+    showMessage(`Subscribed to the creator with username ${username}`);
+  }
+
+  /**
+   * This function checks if the number of subscribers equal to.
+   */
+  async expectNumberOfSubscribersToBe(number: number): Promise<void> {
+    const subscriberCount = await this.page.$eval(
+      '.e2e-test-oppia-total-subscribers',
+      element => element.textContent
+    );
+
+    if (subscriberCount && parseInt(subscriberCount) === number) {
+      showMessage(`Number of subscribers is equal to ${number}`);
+    } else {
+      throw new Error(`Number of subscribers is not equal to ${number}`);
+    }
+  }
+
+  /**
+   * Function for refreshing the page.
+   */
+  async refreshPage(): Promise<void> {
+    await this.page.reload();
+  }
+
+  /**
+   * Function for opening the subscribers tab.
+   */
+  async openSubscribersTab(): Promise<void> {
+    if (this.page.url() !== creatorDashboardUrl) {
+      await this.navigateToCreatorDashboardPage();
+    }
+
+    await this.page.click(subscriberTabButton);
+    await this.page.waitForSelector('.e2e-test-subscription-card');
+  }
+
+  /**
+   * This function checks whether given user is a subscriber or not.
+   */
+  async expectUserToBeASubscriber(username: string): Promise<void> {
+    let truncatedUsername = username;
+    if (username.length > 10) {
+      const ellipsis = '...';
+      truncatedUsername =
+        username.substring(0, 10 - ellipsis.length) + ellipsis;
+    }
+
+    const subscribers = await this.page.$$(subscriberCard);
+
+    if (subscribers.length === 0) {
+      throw new Error(`User "${username}" is not subscribed.`);
+    }
+
+    const subscriberUsername = await subscribers[0].$eval(
+      '.e2e-test-subscription-name',
+      element => (element as HTMLElement).textContent?.trim()
+    );
+
+    if (truncatedUsername === subscriberUsername) {
+      showMessage(`User ${username} is a subscriber`);
+    } else {
+      throw new Error(`User ${username} is not a subscriber`);
+    }
+  }
+
+  /**
+   * Checks whether the exploration with the given title is authored by the creator.
+   */
```

Comment:
Please review the docstring. It should clearly describe what your function is intended to do.
And here I think function name is very much clear to know what is the working of the function.

Reply:
I have updated docstring, and I guess we should keep it. let me know if any change is required.